### PR TITLE
Bilgewater Game Modes

### PR DESCRIPTION
**Pull request by aktai0**
Added Butcher's Bridge (ARAM skin) and Black Market Brawlers (featured game mode) to the CurrentGame, Game, and Match endpoints.

### CODE REVIEW

File: RiotSharp/CurrentGameEndpoint/Converters/GameQueueType.cs [hunk 1]
```diff
@@ -153,11 +153,21 @@ public enum GameQueueType
 		/// <summary>
 		/// Twisted Treeline 6x6 Hexakill games
 		/// </summary>
-		Hexakill = 98,
+      Hexakill = 98,
```
Comment (BenFradet) [inline RiotSharp/CurrentGameEndpoint/Converters/GameQueueType.cs]: Could you please align the code?

File: RiotSharp/CurrentGameEndpoint/Converters/GameQueueTypeConverter.cs [hunk 2]
```diff
@@ -106,10 +106,16 @@ public override void WriteJson(JsonWriter writer, object value, JsonSerializer s
 					break;
 				case GameQueueType.Hexakill:
 					result = "HEXAKILL";
-					break;
+               break;
```
Comment (BenFradet) [inline RiotSharp/CurrentGameEndpoint/Converters/GameQueueTypeConverter.cs]: Same

**Comment by BenFradet** [comment]
LGTM aside from a few formatting quirks.

**Comment by aktai0** [comment]
Fixed the formatting. I didn't realize that those files were using tabs instead of spaces

**Comment by BenFradet** [comment]
Yup, weird indeed, Do you mind converting those files back to spaces?

**Comment by aktai0** [comment]
Done!

File: RiotSharp/CurrentGameEndpoint/Converters/GameQueueType.cs [hunk 3]
```diff
@@ -1,163 +1,173 @@
...
-		
-		/// <summary>
-		/// King Poro games
-		/// </summary>
-		KingPoro5x5 = 300
-	}
+   /// <summary>
+   /// queueType and gameQueueConfigId subtype of CurrentGame API
+   /// </summary>
+   public enum GameQueueType
```
Comment (BenFradet) [inline RiotSharp/CurrentGameEndpoint/Converters/GameQueueType.cs]: The indent should be 4 spaces.

File: RiotSharp/CurrentGameEndpoint/Converters/GameQueueTypeConverter.cs [hunk 4]
```diff
@@ -9,222 +9,234 @@
...
-				case GameQueueType.KingPoro5x5:
-					result = "KING_PORO_5x5";
-					break;
-				default:
-					result = "";
-					break;
-			}
-			serializer.Serialize(writer, result);
-		}
+   class GameQueueTypeConverter : JsonConverter
```
Comment (BenFradet) [inline RiotSharp/CurrentGameEndpoint/Converters/GameQueueTypeConverter.cs]: Same here.

**Comment by BenFradet** [comment]
Sorry to be nitpicky but we do have a 4 spaces indent policy around the project.

**Comment by BenFradet** [comment]
Thanks a lot, merging.

**Pull request merged**